### DISCUSSION TITLE
First steps done, first problems found (With the Printer, not the config) (SV06plus)

### DISCUSSION THREAD

**Discussion by asusdk** [bassamanator/Sovol-SV06-firmware#30] [General]
So I'm done with the Setup, and so far it looks pretty good besides again my alltime-Enemy - the first layer

The Problem I Have right now, is that like on my ender before, the Mesh does not correctly compensate. The First layer is not as perfect as I'd like to.

Mesh:
![image](https://user-images.githubusercontent.com/54312862/232113793-3880de7a-281d-480f-8bd5-b0d30ff862d7.png)
----------------------------------------------------------------------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/54312862/232113829-411875b9-922a-4c87-ba5e-f21ac8da5281.png)

FirstLayer:
![image](https://user-images.githubusercontent.com/54312862/232116200-a2024aaf-f76e-404f-a704-bd10749188d1.png)

![image](https://user-images.githubusercontent.com/54312862/232116687-413fa926-8934-46ca-ac56-2ce979930e40.png)

![image](https://user-images.githubusercontent.com/54312862/232116857-4c149921-a7d3-43e8-a2d9-3e35a3574090.png)

Now I try to straighten the Bed itself by play a bit around with the Screws.

This is not especially a thread to seek help, more a Journal about my first experience with the SovolSv06plus.
If someone has suggestions how to get a better first layer, please comment, otherwise I will update this thread if I reach an acceptable state.

**Comment by superceleron** [comment]
Watch out that gantry level, because it not always does a good job, i think you better check it manually,
I also had the same issue on mine, a side lower than the other and it was the gantry not level correctly on top.
I also mod my bed with silicone spacers.
This is how i have it now, it could be better but i think im ok for the deviation it have now.

<img width="1365" alt="image" src="https://user-images.githubusercontent.com/11303825/232133253-1abf9949-b537-489c-a3c0-96df7f5be4e3.png">

**Comment by asusdk** [comment]
Ok, one Day later now, no matter what I try, the first layer will just not work, so something is off with this printer. Again.... like with the Ender.
Why it is not possible that the manufacturers provide an acceptable aligned Buildplate.... that's really frustrating right now.

I tried to mess with the Screws - did not do anything at all.
I tried to use tape and correct therefore for the crooked Plate. But this doesn't help either.
I Manually tried to Level the Gantry over and over again. A good first layer seems to be impossible.

Very Sad right now.


Printed at least a first Benchy, and so far the Printquality is acceptable, especially if I consider, that i did not have tuned Pressure Advance or retractions right now.
![image](https://user-images.githubusercontent.com/54312862/232229993-5212056c-92b5-4bf3-8b84-cf4a2956b7ba.png)

**Comment by ncoppen** [comment]
New(ish) SV06 owner here, I've also had **a lot** of issues trying to get everything level enough to produce decent first layers. I'll spare you the saga that I've been through so far but what's worked for me in Klipper is manual bed levelling. I used a feeler gauge (0.1mm) a 60ºC bed and a lot of patience trying my best to get a consistent feel across the whole bed. With lightly toasted fingers it produced me quite a different result than the automated bed mesh and so far has given me more consistent first layers.

![Screenshot 60](https://user-images.githubusercontent.com/18281848/232236440-48367cb5-ecda-4b93-a717-b5c8310ea958.png)

![70326383061__27B922AE-0D50-4F3E-96CE-AF600AA5D33B](https://user-images.githubusercontent.com/18281848/232235767-80241030-cdc5-4ac9-948b-356d9eb22a96.jpeg)

Another tweak that I made is greatly halving the first layer speed in the suggested Ellis Super Slicer profiles from 60mm/s to 30mm/s as I found that it was liable to poor line adhesion and lifting. Likely something else for me to tweak but for now I'd rather spend an extra few minutes ensuring a good first layer. Still experiencing some inconsistencies, such as when I print with the infill at 45º I still have a few holes vs printing at 0º pictured above.

**Comment by asusdk** [comment]
Ok, now i am really dont know whats wrong with this printer...

I went to the Hardwarestore, bought some metal-spacers and some thick rubber-spacers.

![image](https://user-images.githubusercontent.com/54312862/232283564-255f9751-c7bd-48cf-9592-ab497219cf88.png)
![image](https://user-images.githubusercontent.com/54312862/232283593-1d221522-5d48-4a76-86ac-5685e75441af.png)

I replaced the original spacers, that the Sv06plus uses each, with 6 metal-spacers + 1 rubber-spacer
I ran 3 x G34 and started finetuning the bed until it's so flat as even possible (at least for me)

![image](https://user-images.githubusercontent.com/54312862/232283675-5569cfe0-5c1b-421a-afc1-51d0bfd11bc9.png)

But still... the first layer is a mess.

What the hell is wrong with this thing. Im really thinking about send this piece of garbage back...

Front left:
![image](https://user-images.githubusercontent.com/54312862/232284005-dca4b1cb-be5f-499e-b57c-03a41ad3861d.png)

Front right:
![image](https://user-images.githubusercontent.com/54312862/232284053-11dfda4b-30a8-448f-a80f-423c7b8f1f31.png)

Back right:
![image](https://user-images.githubusercontent.com/54312862/232284084-04d60292-7c7f-437c-b309-9dd89b3f2eab.png)

Back left:
![image](https://user-images.githubusercontent.com/54312862/232284115-15ab2bd7-cb6c-4907-abf1-b060c7eea8bb.png)

**Comment by superceleron** [comment]
Check your lead screws if they not bent, mine came with the 2 bent and i had lots of issues, until i notice the lead screws spinning and i could see they really wobble like hell, i contacted them and they sent a new pair after that all is ok, of course i needed to do the silicone spacers to the bed since this beds are really warp.
Ho i have the "normal" SV06, so not sure about the Plus.

**Comment by superceleron** [comment]
Ho i forgot, also check your flow, a improper flow also affects a lot the 1º layer.
Play with the flow control slider in Fluidd(or Mainsail), as its laying down the 1º layer and see how it looks.
But for sure that bed is really warp!

**Comment by asusdk** [comment]
I leveled the Bed now manually, and the first layer now works like a charm, really sad, that again, with a Probe i cant get a good layer. It was nearly the Same on my Ender before.

Right now, I tuned just the flow and the retractions.  I can't help it, but I'm not very pleased with the print quality right now. I dont know if its the printer or the Config / Printprofile.

![image](https://user-images.githubusercontent.com/54312862/232307740-587aafeb-5388-4487-af61-c0850d0c0609.png)
![image](https://user-images.githubusercontent.com/54312862/232307757-a18ee5a9-e8a1-49ec-9328-b7c32a3c74d7.png)
![image](https://user-images.githubusercontent.com/54312862/232307773-060e796d-028a-4961-adc0-c40936cbd59f.png)
![image](https://user-images.githubusercontent.com/54312862/232307790-71ef7416-aa0f-4e8f-8a8c-d54b7e013304.png)


On the Pictures it looks like the flow is to low, so i raised it. Printquality got a bit better now, i Lowered Acceleration in the Printprofile at all occasions of 3000  down to 2000, also i slowed down the print a bit.

Printing a Calibcube again to assure that flow is good now, afterwards ill finetune retraction and pressura Advance.

As before i will update this thread. But so or so, this is the last cheap printer i bought. period.

**Reply by bassamanator** [reply]
@asusdk That print quality is very acceptable. You simply need to do some tuning. Also, the single layers in orange that you printed above were okay too, though the z_offset was a bit off, you simply needed to increase the gap between the nozzle and the bed. You do not have a lemon here. 

This is what buying a budget printer is all about: you need to get it up to speed, it's all part of the process, and for some of us, it's part of the fun. It seems you were looking for the Bambu Labs 3d printer experience, but you can't get that in this price range.

I have to say your approach to fixing the bed levelling with those spacers was unique. Please consider creating a short How-To in the discussions. I think this method is simpler than the silicon spacer mod.

**Comment by superceleron** [comment]
Go back to the basics, flash the stock fw and try it if you still getting issues maybe you got a lemon, if so then time to think on sending it back!
